### PR TITLE
Fix markdown formatting

### PR DESCRIPTION
**Pull request by BILLthebuilder**
This Pull request fixes some markdown formatting details that may have been overlooked.

### CODE REVIEW

**Comment by sebmck** [comment]
Any reason the dollar signs were removed from only some of the commands?

**Comment by BILLthebuilder** [comment]
It was probably because of [this](https://github.com/updownpress/markdown-lint/blob/master/rules/014-commands-show-output.md)
I have markdown-lint configured, they were removed by default

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -8,10 +8,10 @@ Our [Discord server](https://discord.gg/9WxHa5d) is open for help and more adhoc
 
 Getting started with developing Rome is as easy as three commands. You will need Node v12 or above.
 
-```
+```bash
 $ git clone https://github.com/facebookexperimental/rome
 $ cd rome
```
Comment (tomByrer) [inline .github/CONTRIBUTING.md]: Might be more clear to remove all `$` from lines to help with copy-paste-coding.

IMHO the only time `$` should be used if for a 'demo` of output.

**Comment by tomByrer** [comment]
I like the addition of `bash` to the code blocks!

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (BILLthebuilder) [reply .github/CONTRIBUTING.md]: @sebmck what do you think? Should I remove all the dollar signs in the commands?

**Comment by sebmck** [comment]
Happy to have all the dollars from all the removed! Once you make that change I can merge. Thank you!

**Comment by BILLthebuilder** [comment]
> Happy to have all the dollars from all the removed! Once you make that change I can merge. Thank you!

Its done now

**Comment by sebmck** [comment]
Thank you so much!

**Pull request merged**